### ISSUE TITLE
R/a/dio API Docs

### ISSUE THREAD

**Issue by ameliaikeda** [R-a-dio/Android-app#53]
Incoming when I get home, I'm sitting in a café typing out the markdown docs on my phone. The API will be at https://api.r-a-d.io/api/chitose. The https should be mandatory, and I'll give the App an api key to use (public, sha256) that will rate limit apps (basically you can make a call every 2 seconds and be fine).

Also, load all assets over TLS. The new site will stop proxy-mangling on mobile connections, but it's slightly far away.

tl;dr this is for me until I yell in IRC.

Following this, we should be fine for a release.
